### PR TITLE
Velocitas offline

### PR DESCRIPTION
**Pull request by BjoernAtBosch**
venv-based execution of Python tooling could look like this.

Issues:
* Shell scripts for venv exection in root dir and per pkg repo
* quite a lot of "boiler plate" stuff in the manifest to setup+call the venvs

Question is, if we (could) have some CLI magic improving that?

### CODE REVIEW

File: setup/src/common/scripts/onCreateCommand.sh [hunk 1]
```diff
@@ -21,6 +21,8 @@ sudo chown -R $(whoami) $HOME
 if [[ -z "${VELOCITAS_OFFLINE}" ]]; then
     .devcontainer/scripts/configure-codespaces.sh
     .devcontainer/scripts/upgrade-cli.sh
+elif [[ -x .devcontainer/scripts/local-setup.sh ]]; then
+    .devcontainer/scripts/local-setup.sh
```
Comment (erikbosch) [inline setup/src/common/scripts/onCreateCommand.sh:25]: Is this change unrelated, or do we anywhere have a description on when/how/why you sometimes may use a `local-setup.sh` file and allowed content (like no actions requiring online connectivity?)

Comment (BjoernAtBosch) [reply setup/src/common/scripts/onCreateCommand.sh:25]: It's related. With that I wanted to offer a possibility to do repo specific set-up for the offline variant. Problem is, that all the other scripts are "CLI maintained".

Comment (BjoernAtBosch) [reply setup/src/common/scripts/onCreateCommand.sh:25]: Remark: I will probably change the location of this call. It's currently a bit strange from log output's perspective.

**Comment by erikbosch** [comment]
Just a thought - does this change affect our license generation in any way? I.e. if we have a direct dependency to A and B, and both in turn have a dependency to C but different versions of C, will our generated NOTICE 3rd party file include both versions, or only one of them. Do not know if it is a realistic example.

**Comment by BjoernAtBosch** [comment]
> Just a thought - does this change affect our license generation in any way? I.e. if we have a direct dependency to A and B, and both in turn have a dependency to C but different versions of C, will our generated NOTICE 3rd party file include both versions, or only one of them. Do not know if it is a realistic example.

Good point. I even don't know, how it is currently working, because of the requirements files distributed over the component folders ....

**Review by erikbosch — DISMISSED** [review]
Looks good to me.
No local checks performed.

**Review by erikbosch — APPROVED** [review]
LGTM

**Pull request merged**